api/src/main/java/org/pmiops/workbench/actionaudit/targetproperties/BypassTimeTargetProperty.kt

```diff
@@ -2,10 +2,10 @@ package org.pmiops.workbench.actionaudit.targetproperties
 
 enum class BypassTimeTargetProperty
 constructor(override val propertyName: String) : SimpleTargetProperty {
-    DATA_USE_AGREEMENT_BYPASS_TIME("data_use_agreement_bypass_time"),
-    COMPLIANCE_TRAINING_BYPASS_TIME("compliance_training_time"),
-    CT_COMPLIANCE_TRAINING_BYPASS_TIME("ct_compliance_training_time"),
-    ERA_COMMONS_BYPASS_TIME("era_commons_bypass_time"),
-    TWO_FACTOR_AUTH_BYPASS_TIME("two_factor_auth_bypass_time"),
-    RAS_LINK_LOGIN_GOV("ras_link_login_gov_bypass_time")
+    DATA_USER_CODE_OF_CONDUCT("data_use_agreement_bypass_time"),
```

Comment:
changing the **strings** would mean changing the recorded audit values.  That needs more thought.

Reply:
We're free to rename the enum values though - so now they match AccessModuleName